### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by cs-miller**
I removed references to installing the beta, and replaced them with links to the App Store and Play Store.

### CODE REVIEW

**Comment by dkwl93** [comment]
LGTM - Would like @Proyoyo to review properly though.

**Comment by Proyoyo** [comment]
@cs-miller There's some padding on the Google Play badge. Should we put them on different line to make it look better?

screen shot 2018-03-07 at 11 09 48 am

File: README.md [hunk 1]
```diff
@@ -1,73 +1,39 @@
...
 * Follow the official [React Native guide](https://facebook.github.io/react-native/docs/getting-started.html) to setup your environment.
 * Clone the repo
 * `cd` into local folder and do a `yarn install` to install dependencies (or use `npm install`)
 * run `react-native run-ios` or `react-native run-android`
 
-Notes
-----
+## Notes
+
 * Still a work in progress
```
Comment (Proyoyo) [inline README.md]: We can remove this section for now.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,39 @@
...
-<img src="https://i.imgur.com/ejopAVs.png" height="640" width="360"/>
-
-4. Install the downloaded apk.
-
-TestFlight Download Instruction
-----
-1. Open the invitation email from TestFlight and click "View in TestFlight".
-<img src="https://i.imgur.com/AEUduMb.png" height="239" width="318"/>
+<a href="https://itunes.apple.com/us/app/opendota/id1354762555?mt=8"><img alt='Download on the App Store' src='https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/appstore-lrg.svg' width="135"/></a>
+<a href='https://play.google.com/store/apps/details?id=com.opendota.mobile&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1' ><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png' width="135"/></a>
```
Comment (Proyoyo) [inline README.md]: Can we add the Google Play badge to a different line? It looks janky because it has padding while Apple's doesn't.

**Review by Proyoyo — CHANGES_REQUESTED** [review]

**Review by Proyoyo — APPROVED** [review]

**Pull request merged**